release/gcb/gcb_lib.sh

```diff
@@ -20,6 +20,7 @@
 #shellcheck disable=SC2164
 SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
 # shellcheck source=release/gcb/docker_tag_push_lib.sh
+#shellcheck disable=SC1091
```

Comment:
Should either remove line 22 or remove this line as they are both trying to fix the same thing